x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/formula_help.tsx

```diff
@@ -314,7 +334,12 @@ export function getFunctionSignatureLabel(
   }
   if (operationDefinitionMap[name]) {
     const def = operationDefinitionMap[name];
-    return `${name}(${def.documentation?.signature})`; // ${firstParam ? firstParam.label : ''})`;
+    let extraArgs = '';
+    if (def.filterable) {
+      extraArgs += hasFunctionFieldArgument(name) || 'operationParams' in def ? ',' : '';
+      extraArgs += '[kql]?: string, [lucene]?: string';
```

Comment:
@dej611 We probably don't need i18n on the words kql/lucene, but we probably do need it for the word `string`.